src/.pre-commit-config.yaml.jinja

```diff
@@ -37,7 +37,10 @@
 {%- endif %}
 
 {#- Older versions that differ a lot have their own hardcoded templates for readability #}
-{%- if odoo_version < 14 %}
+{%- if odoo_version < 13 %}
+  {%- include "version-specific/mqt-compat/.pre-commit-config.yaml.jinja" %}
+
+{%- elif odoo_version < 14 %}
```

Comment:
Is it possible that this line breaks updates for version < 13 as there is only one `version-specific/13.0` folder... ?

 @carmenbianca @sbidoul

Reply:
Maybe it's a string/float problem? It expects a float where there should be a string, or vice versa? Frankly I'm not sure. I wish I knew how to help.

Reply:
You can try replacing `odoo_version` with `odoo_version|float` or `odoo_version|int` in these checks. Just in case you're passing the version as a string for some reason.

Also make sure you're using the latest copier.